### PR TITLE
Mission styling

### PR DESCRIPTION
**Pull request by ryanmadamba**
This PR adds styling for the Mission Overview page.

### CODE REVIEW

File: src/Components/MTaskOverview.js [hunk 1]
```diff
@@ -34,9 +34,12 @@ function MTaskOverview(props) {
...
-         <h2>Points: {props.task.points}</h2>
-         <h2>Due: {props.task.dueDate}</h2>
+         <div className="line"></div>
+         <h3>Task Description:</h3>
+         <h4>{props.task.instructions}</h4>
+         <h4>Points: {props.task.points}</h4>
+         <h4>Due: {props.task.dueDate}</h4>
+         <div className="line"></div>
          <h2>{mapRequirementsToCheck(props.task.requirements)}</h2>
          <div  className="start">
```
Comment (ConnorBoulais) [inline src/Components/MTaskOverview.js:44]: Task Overview looks a lot better

File: src/Components/Mission.js [hunk 2]
```diff
@@ -68,28 +70,52 @@ function Mission(props) {
...
+            </div>  
+          </div>
+          
+            <div className="preview">
+              <MTaskOverview task={focusedTask}/>
+            </div>
+          
+        </div>
+      </div>
+    </div>
```
Comment (ConnorBoulais) [inline src/Components/Mission.js:102]: Great job displaying the background map, it looks awesome

File: src/Components/Mission.js [hunk 3]
```diff
@@ -68,28 +70,52 @@ function Mission(props) {
...
+  );
+}
+
+export default Mission;
+
+/*
+<div className="row">
```
Comment (tddumont) [inline src/Components/Mission.js:109]: Might be good to get rid of this commented code, unless it is being saved for some reason.

File: src/Components/Mission.css [hunk 4]
```diff
@@ -1,95 +1,231 @@
-.SubMission {
-    position: relative;
-    background-color: #F21E1D;
+.background {
```
Comment (tddumont) [inline src/Components/Mission.css:1]: A lot of great styling work here!

**Review by ConnorBoulais — APPROVED** [review]
Awesome, great job styling this component

**Review by tddumont — APPROVED** [review]
Great work!

**Pull request merged**